rasa_core/training/interactive.py

```diff
@@ -1005,6 +1005,7 @@ def _validate_user_text(latest_message: Dict[Text, Any],
     else:
         question = questionary.confirm(
             "Is the NLU classification for '{}' with intent '{}' correct?"
+            "\n    AND are all occurring entities labeled correctly?"
```

Comment:
could we maybe rephrase this so it's a little more intuitive? Right now the sentences together don't make much sense. So, phrasing it like "Is the NLU classification for '{}': intent - {} and entities: {} correct?". Where entities would be none in the case of none of them